src/timestepper.cpp

```diff
@@ -533,7 +542,8 @@ void TimeStepper::energyPenaltyIntegral_diff(double time, double penaltybar, Vec
 
 void TimeStepper::evolveBWD(const double tstart, const double tstop, const Vec x_stop, Vec x_adj, Vec grad, bool compute_gradient){}
 
-ExplEuler::ExplEuler(MapParam config, MasterEq* mastereq_, int ntime_, double dt_, Output* output_, bool storeFWD_, MPI_Comm comm_time_) : TimeStepper(config, mastereq_, ntime_, dt_, output_, storeFWD_, comm_time_) {
+ExplEuler::ExplEuler(MapParam config, MasterEq* mastereq_, int ntime_global_, int ntime_, double dt_, Output* output_, bool storeFWD_, MPI_Comm comm_time_)
+  : TimeStepper(config, mastereq_, ntime_global_, ntime_, dt_, output_, storeFWD_, comm_time_) {
```

Comment:
Please avoid making white space changes

Reply:
can you elaborate more on this?

Reply:
I find it confusing with modifications that only change the formatting and not the functionality of the code. My approach is to let VS-code take care of the formatting.

Reply:
Revoked the line change.